config/knative/addressable.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: discovery.knative.dev/v1alpha1
+kind: DuckType
+metadata:
+  name: addressables.duck.knative.dev
+spec:
+  selectors:
+    - labelSelector: "duck.knative.dev/addressable=true"
+  names:
+    name: "addressable"
+    plural: "addressables"
+    singular: "addressable"
+  versions:
+    - name: "v1"
+      refs:
+        - group: "duck.knative.dev"
```

Comment:
For more examples, I would add a second one to use the `resource` field too so we can exercise that side of the code too.